server/lib/src/idm/ldap.rs

```diff
@@ -643,15 +642,51 @@ mod tests {
         let pce = UnixPasswordChangeEvent::new_internal(UUID_ADMIN, TEST_PASSWORD);
 
         assert!(idms_prox_write.set_unix_account_password(&pce).is_ok());
-        assert!(idms_prox_write.commit().is_ok());
+        assert!(idms_prox_write.commit().is_ok()); // Committing all configs
 
-        let anon_t = ldaps.do_bind(idms, "", "").await.unwrap().unwrap();
+        // Test for optional use of POSIX password for bind
+        // Empty UNIX_PW bind, should allow :
+        let admin_t = ldaps
+            .do_bind(idms, "admin", TEST_PASSWORD)
+            .await
+            .unwrap()
+            .unwrap();
+        assert!(admin_t.effective_session == LdapSession::UnixBind(UUID_ADMIN));
+        let admin_t = ldaps
```

Comment:
Why do this twice?

Reply:
they are on different DNs, just wanted to include those in test suites as well....can remove it